src/renderer/react/app/nodes/manager/editNodeModal/EditNodeModal.jsx

```diff
@@ -54,10 +54,10 @@ class EditNodeModal extends React.PureComponent {
     /** From createForm / Form. */
     form: formShape.isRequired,
     /** Parent ID available options. */
-    parentIdOptions: PropTypes.shape({
+    parentIdOptions: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
what are the parent id options ?

Reply:
It's a selector that compute all available parents for a node, so you can move it in the tree without causing infinite recursion.
I made a mistake with its proptypes that i'm correcting in this PR